src/ontology/mondo-edit.obo

```diff
@@ -312181,20 +312189,9 @@ relationship: seeAlso https://github.com/monarch-initiative/mondo/issues/4069
 
 [Term]
 id: MONDO:0016050
-name: thiamine-responsive encephalopathy
-def: "Thiamine-responsive encephalopathy is a Wernicke-like encephalopathy characterized by seizures responsive to high doses of thiamine." [Orphanet:199348]
-comment: Reason: duplicate. This will be merged with MONDO:0011841
-subset: obsoletion_candidate
-subset: ordo_disease {source="Orphanet:199348"}
-xref: OMIM:607483 {source="MONDO:subClassOf", source="MONDO:relatedTo", source="ORDO:199348/ntbt", source="Orphanet:199348"}
-xref: Orphanet:199348 {source="MONDO:equivalentTo"}
-xref: SCTID:723557004 {source="MONDO:equivalentTo"}
-is_a: MONDO:0005527 {source="https://github.com/monarch-initiative/mondo/issues/1520", source="https://orcid.org/0000-0001-5208-3432"} ! toxic encephalopathy
-relationship: disease_responds_to CHEBI:26948 ! vitamin B1
-relationship: excluded_subClassOf MONDO:0015653 {source="Orphanet:199348"} ! monogenic epilepsy
-relationship: seeAlso https://github.com/monarch-initiative/mondo/issues/4205
-property_value: confidence "1.1497584541062804" xsd:double
-property_value: IAO:0006012 "2022-03-01" xsd:string
+name: obsolete thiamine-responsive encephalopathy
+is_obsolete: true
+replaced_by: MONDO:0011841
```

Comment:
could you please a link to the seeAlso https://github.com/monarch-initiative/mondo/issues/4205
and the annotation has obsolescence reason terms merged

thank you!